### PR TITLE
libtiff: build static libs when shared=False

### PR DESCRIPTION
**Pull request by SpaceIm**
closes https://github.com/conan-io/conan-center-index/issues/14206

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libtiff/all/conanfile.py [hunk 1]
```diff
@@ -70,19 +70,9 @@ def config_options(self):
 
     def configure(self):
         if self.options.shared:
-            try:
-                del self.options.fPIC
-            except Exception:
-                pass
-        if not self.options.cxx:
```
Comment (stevn) [inline recipes/libtiff/all/conanfile.py:77]: @SpaceIm I can't tell at the moment if removing this line is OK or not... Wouldn't you only want to remove compiler.cppstd and compiler.libcxx if the cxx option is not set?

Comment (SpaceIm) [reply recipes/libtiff/all/conanfile.py:77]: oops correct, I'll fix that

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by SpaceIm** [comment]
FYI, I've proposed a hook to check existence of at least one static lib when shared=False: https://github.com/conan-io/hooks/pull/455

**Review by AndreyMlashkin — APPROVED** [review]

**Pull request merged**